CFHT references

mpaillassa opened the pull request:
Here are updates concerning CFHT #28.

I completed the filters to u, g, r, i, z. Most references come from the [CFHT-MegaCam general information page](https://www.cfht.hawaii.edu/Instruments/Imaging/Megacam/generalinformation.html) but the `psf_fwhm` is from [CFHTLS](https://www.cfht.hawaii.edu/Science/CFHTLS/T0007/CFHTLS_T0007-TechnicalDocumentation.pdf) because I could not find it elsewhere.
So I was wondering if we should prefer general CFHT or CFHTLS parameters ? For instance, we could set CFHTLS exposure times instead of the current 300s. But if we go for this I guess we should change the survey name to CFHTLS. What do you think ? 
I ping @aguinot as you surely have an opinion about this !

aboucaud commented:
Thanks for taking this on.
I guess you're right. We decided to go for the survey and not the instrument in the end, so in that regard, CFHTLS would probably make more sense.

@aguinot ?

aguinot commented:
I think it is easier to attach those values to a specific survey, especially for the psf size. The CFHT can observe with a seeing of 2" if you don't care about quality and it is more atmospheric dependent than  instrument (for ground based survey). So this sounds good to me.

mpaillassa commented:
Thanks ! 
I just changed the exposure times to that of CFHTLS and renamed accordingly. I guess it is now ok except if you see something wrong @aboucaud @aguinot

aboucaud reviewed (approved):
Looking good !
